blueocean-bitbucket-pipeline/pom.xml

```diff
@@ -54,6 +54,11 @@
       <artifactId>httpmime</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
this seems to have moved around - for no reason I assume? (just obsessing over pom changes...)

Reply:
Just moving it above the "test dependencies" heading...